README.md

```diff
@@ -85,11 +85,8 @@ while True:
     if msg is None:
         continue
     if msg.error():
-        if msg.error().code() == KafkaError._PARTITION_EOF:
-            continue
-        else:
-            print(msg.error())
-            break
+        print(msg.error())
+        break
```

Comment:
Let's change this to a continue while we are at it, errors are typically temporary.
Change to:
```
print("Consumer error: {}".format(msg.error()))
continue
```

Reply:
ack